.github/workflows/module-test.yml

```diff
@@ -16,7 +16,6 @@ jobs:
         target:
           [
             { os: centos, version: 8, package-type: RPM },
-            { os: debian, version: 10, package-type: DEB },
```

Comment:
>{ os: debian, version: 10, package-type: DEB }, [](http://example.com/codeflow?start=12&length=47)

Debian 10 tests do actually work. We do not want to remove from CI any tests that work and may reveal regressions at PR time. 

This PR would make sense if we decide with MC to no longer provide support for past EOL distros and route MC calls for that to v1 implementation via the 'multi-MOF' method. 

Until then, we need to cover in CI as many distros as possible.

Suggest that we postpone this PR until then.

In meantime, we do want to complete the list here for distros that are not past EOL. We are still missing Azure Linux 3 and Amazon Linux 2, for example.

Reply:
i know they work, but this is about removing distros that are EOL. We can add the Debian 10 to our list of distros to validate manually.

Good point on the Azure Linux 3 and Amazon Linux 2 onboarding. I forgot about those, ill make surew to include them in the list